### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by cliserkad**
Replaces links to the release files. Updates the versions of Java and LWJGL. Adds build commands.

### CODE REVIEW

**Review by CiroZDP — CHANGES_REQUESTED** [review]
Alright but the link to the download isn't working.

File: .github/README.md [hunk 1]
```diff
@@ -73,22 +73,23 @@ I hope that you can be part of that community forking the repository or simply,
 
 ---
 
-## ☕ Downloading
-We already have prebuilt jars. They can be downloaded [here](https://github.com/OpenCraft-Studios/OpenCraft/actions).
+## ☕ Downloads
```
[the remark this answers is not in the document]
Comment (CiroZDP) [reply .github/README.md:76]: Thinking about it, some people would like to directly download the JARS

**Comment by cliserkad** [comment]
> Alright but the link to the download isn't working.

Which one? Both the .bat and .jar download links work for me

**Comment by CiroZDP** [comment]
> > Alright but the link to the download isn't working.
> 
> Which one? Both the .bat and .jar download links work for me

At the top of the readme, there's a link, in that section, there is a link called "Downloading". That doesn't works anymore because you changed the name of the section.

**Comment by CiroZDP** [comment]
> > Alright but the link to the download isn't working.
> 
> Which one? Both the .bat and .jar download links work for me

To be exactly:
image

**Comment by CiroZDP** [comment]
Awesome!

**Review by CiroZDP — APPROVED** [review]

File: .github/README.md [hunk 2]
```diff
@@ -73,22 +73,23 @@ I hope that you can be part of that community forking the repository or simply,
...
-   > ```powershell
-   > iex (iwr -Uri https://opencraft-studios.github.io/install.ps1)
-   > ```
-   >
+## 🚧 Building
 
-   </details>
+```bash
+git clone https://github.com/OpenCraft-Studios/OpenCraft
+cd OpenCraft
```
[the remark this answers is not in the document]
Comment (cliserkad) [reply .github/README.md:89]: I expect people who are building the project to manually run the commands.

[the remark this answers is not in the document]
Comment (CiroZDP) [reply .github/README.md:89]: Okay. It's fine if you want.

File: .github/README.md [hunk 3]
```diff
@@ -46,7 +46,7 @@
 <br />
 
 ## ✅ Getting started
-**OpenCraft** is a game programmed in Java, it has the same motor (LWJGL 2 + Java 17) that Minecraft.
+**OpenCraft** is a game programmed in Java, using the same tools (LWJGL 3 + Java 21) as Minecraft.
```
[the remark this answers is not in the document]
Comment (cliserkad) [reply .github/README.md:49]: You have to program all of the graphics, input handling, and game update loop yourself. So I wouldn't say it's an engine.

[the remark this answers is not in the document]
Comment (CiroZDP) [reply .github/README.md:49]: I don't get into much technical explanations but if you want to call it just a motor go ahead.

**Pull request merged**